airflow/config_templates/config.yml

```diff
@@ -375,6 +375,32 @@ core:
       type: string
       example: "path.to.CustomXCom"
       default: "airflow.models.xcom.BaseXCom"
+    xcom_objectstorage_path:
```

Comment:
This should be moved to provider.yaml, since the storage backend is now there.

Reply:
Ah okay, didn't know that was possible.

Reply:
That was my `vacation` pet-project for 2.7 release :)